models/stg_google_play__store_performance_country.sql

```diff
@@ -0,0 +1,36 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_google_play__store_performance_country_tmp') }}
+
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_google_play__store_performance_country_tmp')),
+                staging_columns=get_stats_store_performance_country_columns()
+            )
+        }}
+        
+    from base
+),
+
+final as (
+    
+    select 
+        date as date_day,
+        country_region,
+        package_name,
+        -- just to batch NULL countries into one pile
```

Comment:
I feel like this is kind of a floating comment here, I'm uncertain what it is referring to? And is this `null` batching different than the `null` batching comment you have for group bys?

Reply:
no it's the same `null` batching! would having the same comments as the other models help clarify that?

Reply:
To recap our convo - decided to add to decisionlog as it touches many models

Reply:
To recap our convo - decided to add to decisionlog as it touches many models